scripts/installer/windows/scripts/pfsvc.py

```diff
@@ -48,6 +48,9 @@ def init():
         servicemanager.Initialize()
         servicemanager.PrepareToHostSingle(PromptFlowServiceFramework)
         servicemanager.StartServiceCtrlDispatcher()
+    elif len(sys.argv) == 2 and sys.argv[1] == "manual":
```

Comment:
this command looks strange, but let's use this.

Reply:
fix it